composer/algorithms/swa/README.md

```diff
@@ -0,0 +1,71 @@
+# 🧩 Stochastic Weight Averaging
+
+[\[How to Use\]](#how-to-use) - [\[Suggested Hyperparameters\]](#suggested-hyperparameters) - [\[Attribution\]](#attribution)
+
+ `Computer Vision`, `Natural Language Processing`
+
+Stochastic Weight Averaging (SWA) maintains a running average of the weights towards the end of training. This leads to better generalization than conventional training.
+
+| ![SWA](https://storage.googleapis.com/docs.mosaicml.com/images/methods/swa.png) |
+|:--|
+|*Visualization of SWA from an extensive [PyTorch blogpost about SWA](https://pytorch.org/blog/stochastic-weight-averaging-in-pytorch/).*|
+
+
+## How to Use
+
+### Composer Trainer
+
+<!-- TODO: Address timeouts -->
+<!--pytest-codeblocks:skip-->
```

Comment:
Can you enable this test? See https://github.com/mosaicml/composer/pull/1279 which should be merged any sec for a few examples

Reply:
Lmk if this looks right